Userland/Libraries/LibTest/TestSuite.cpp

```diff
@@ -5,14 +5,26 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include "AK/Format.h"
+#include "AK/JsonArray.h"
+#include "AK/JsonObject.h"
+#include "AK/JsonValue.h"
+#include "LibCore/IODevice.h"
 #include <LibTest/Macros.h> // intentionally first -- we redefine VERIFY and friends in here
```

Comment:
Very useful comment, yes.

Reply:
it's correct now, the funky includes have been removed :)